content/en/docs/ambient/architecture/index.md

```diff
@@ -33,9 +81,8 @@ However, depending on the destination's capabilities, different behavior will oc
 If the destination is also captured, or otherwise has Istio proxy capabilities (such as a sidecar), the request will be upgraded to an encrypted {{< gloss "HBONE" >}}HBONE tunnel{{< /gloss >}}.
 If the destination has a waypoint proxy, in addition to being upgraded to HBONE, the request will instead be forwarded to that waypoint.
 
-Note that in the case of a request to a `Service`, a specific endpoint will be selected to determine if it has a waypoint.
-However, if it *has* a waypoint, the request will be sent with a target destination of the `Service`, not the selected endpoint.
-This allows the waypoint to apply service-oriented policies to the traffic.
+Note that in the case of a request to a `Service`, if the service *has* a waypoint, the request will be sent to its waypoint to enforce service-oriented policies to the traffic.
+Similarly, in the case of a request to a `Pod` IP, if the pod *has* a waypoint, the request will be sent to its waypoint to enforce pod-oriented policies to the traffic.
```

Comment:
Same as above comment. Especially since the waypoint capture scope defaults to service unless specified otherwise. I'm not sure that `use-waypoint` will work as intended if the waypoint only handles service traffic.

Reply:
Agreed, if the waypoint capture scope isn't workload, i'd think the label of use-waypoint on the pod won't be useful.

Reply:
Updated to make this clear, PTAL @MorrisLaw !!

Reply:
Oops, I am so sorry!!! @MorrisLaw